pkg/daemon/agent/flexvolume/controller.go

```diff
@@ -250,11 +250,17 @@ func (c *Controller) parseClusterName(storageClassName string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	clusterName, ok := sc.Parameters["clusterName"]
+	clusterName, ok := sc.Parameters["clusterNamespace"]
```

Comment:
Updated the variables.
I think the name of the function should be updated and in https://github.com/rook/rook/blob/master/pkg/daemon/agent/flexvolume/types.go#L43 it will be good to change ClusterName to ClusterNamespace and make other changes accordingly. Shall i proceed with this?